src/tests/ftest/control/dmg_pool_evict.py

```diff
@@ -54,17 +54,15 @@ def test_dmg_pool_evict(self):
             self.fail(
                 "daos pool list-cont with first pool failed after pool evict!")
 
-        # Call daos pool list-cont on the evicted pool. It should fail with
-        # -1012.
+        # Call daos pool list-cont on the evicted pool. It should succeed.
         try:
             daos_cmd.pool_list_cont(
                 pool=self.pool[1].uuid)
-            self.fail(
-                "daos pool list-cont with second pool succeeded after pool " +
-                "evict!")
-        except CommandFailure:
             self.log.info(
-                "daos pool list-cont with second pool failed as expected")
+                "daos pool list-cont with second pool succeeded as expected")
+        except CommandFailure:
+            self.fail(
+                "daos pool list-cont with first pool failed after pool evict!")
```

Comment:
```suggestion
                "daos pool list-cont with second pool failed after pool evict!")
```
self.pool[1], so second pool?

Reply:
Thanks for the review and comment, please see the new commit.